cli/src/action/login/oauth/callback.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright 2018-2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::collections::HashMap;
+
+use tiny_http::{Header, Method, Request, Response, Server};
+
+use super::error::OAuth2CallbackError;
+
+pub struct OAuth2Callback {
```

Comment:
There should be some documentation for this module and/or this struct, maybe just what's in the commit message. It may not be immediately clear what this is for when other people are looking at it in the future.